packages/sui-widget-embedder/compiler/production.js

```diff
@@ -1,6 +1,8 @@
 const path = require('path')
 const webpack = require('webpack')
 const prodConfig = require('@s-ui/bundler/webpack.config.prod')
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
```

Comment:
hey! s
hould we install it in the `package.json` file or it's already available?